application/src/utils/i18next.ts

```diff
@@ -18,6 +18,7 @@ i18n
     },
     // keySeparator: false,
     // we use keys!
+    //  react: { useSuspense: false }
```

Comment:
in case we don't use it - let's remove it

Reply:
fixed